src/inabox/inabox-resources.js

```diff
@@ -157,6 +157,11 @@ export class InaboxResources {
     return this.firstPassDone_.promise;
   }
 
+  /** @override */
+  isExperimentOn(unusedName) {
```

Comment:
This is a bit concerning. IMHO, if we want to do this, we should just name it more explicit: `isIntersectionExperimentOn()`. I really don't want to see other components start using this method for wrong reasons.